src/util/sandbox/test_sandbox.c

```diff
@@ -0,0 +1,396 @@
+#ifndef FD_HAS_SANDBOX
+#define FD_HAS_SANDBOX
+#endif
+
+#ifdef FD_HAS_SANDBOX
```

Comment:
Likewise, this is probably unnecessary given the tweaks above and perhap usage of FD_HAS_SANDBOX in the Local.mk (such that the test is only built on targets where it is necessary).